asset-properties-widget/component/asset-properties-config/asset-property-selector/asset-property-selector.component.ts

```diff
@@ -44,7 +44,7 @@ export class AssetPropertiesSelectorComponent {
   ) {}
 
   async ngOnChanges(changes: IManagedObject): Promise<void> {
```

Comment:
address lint issue here by adding the OnChanges interface in the class definition

Reply:
incorporated